pages/instructorAssessmentManualGrading/instanceQuestion/instanceQuestion.js

```diff
@@ -132,6 +197,23 @@ router.post(
           res.locals.instance_question.id
         )
       );
+    } else if (req.body.__action === 'modify_rubric_settings') {
+      const rubric_items = Object.values(qs.parse(qs.stringify(req.body)).rubric_item || {});
```

Comment:
Why is `req.body` round-tripped through a stringify and parse?

Reply:
`req.body` does not handle inputs that represent objects (e.g., `rubric_item[points]`), it just creates a flat object. `qs` builds an object for that. If you have a better suggestion to handle these I'm open.